### ISSUE TITLE
[Bug]: applications updates

### ISSUE THREAD

**Issue by grrosminet** [nextcloud/server#32311]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When there is only one app to update, the header of the apps page indicates that "all apps are up to date" ("Toutes les applications sont à jour" in french) whereas on the first row we ca see there is an app which is not up to date.

Whene there are multiple update, an "update all" button is well displayed as expected.

### Steps to reproduce

1. go on "applications" page when there is only one app to update

### Expected behavior

It should indicate there is one app update pending

### Installation method

Manual installation

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 7.4

### Web server

Apache (supported)

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 22.2.3 to 22.2.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "22.2.7.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "loglevel": "2",
        "maintenance": false,
        "theme": "",
        "trusted_domains": [
            "cloud.l*******s.net",
            "cloud.c*****s.tf"
        ],
        "updatechecker": true,
        "mail_smtpmode": "sendmail",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "trashbin_retention_obligation": "auto",
        "appstore.experimental.enabled": true,
        "asset-pipeline.enabled": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "overwrite.cli.url": "https:\/\/cloud.c*****s.tf",
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "mail_sendmailmode": "smtp",
        "mail_smtpsecure": "ssl",
        "app_install_overwrite": [
            "files_opds",
            "files_readmemd",
            "spreed"
        ],
        "default_phone_region": "FR",
        "has_rebuilt_cache": true
    }
}
```

### List of activated Apps

```shell
Enabled:
  - accessibility: 1.8.0
  - apporder: 0.15.0
  - bruteforcesettings: 2.4.0
  - calendar: 3.3.0
  - circles: 22.3.1
  - cloud_federation_api: 1.5.0
  - comments: 1.12.0
  - contacts: 4.1.0
  - contactsinteraction: 1.3.0
  - dav: 1.20.0
  - extract: 1.3.5
  - federatedfilesharing: 1.12.0
  - federation: 1.12.0
  - files: 1.17.0
  - files_pdfviewer: 2.3.1
  - files_rightclick: 1.1.0
  - files_sharing: 1.14.0
  - files_trashbin: 1.12.0
  - files_versions: 1.15.0
  - files_videoplayer: 1.11.0
  - firstrunwizard: 2.11.0
  - keeweb: 0.6.8
  - logreader: 2.7.0
  - lookup_server_connector: 1.10.0
  - maps: 0.1.10
  - metadata: 0.16.0
  - music: 1.5.2
  - news: 18.0.1
  - nextcloud_announcements: 1.11.0
  - notifications: 2.10.1
  - oauth2: 1.10.0
  - password_policy: 1.12.0
  - photos: 1.4.0
  - privacy: 1.6.0
  - provisioning_api: 1.12.0
  - rainloop: 7.2.5
  - recommendations: 1.1.0
  - serverinfo: 1.12.0
  - settings: 1.4.0
  - sharebymail: 1.12.0
  - spreed: 12.2.5
  - support: 1.5.0
  - systemtags: 1.12.0
  - tasks: 0.14.4
  - text: 3.3.1
  - theming: 1.13.0
  - twofactor_backupcodes: 1.11.0
  - updatenotification: 1.12.0
  - user_status: 1.2.0
  - viewer: 1.6.0
  - weather_status: 1.2.0
  - workflowengine: 2.4.0
Disabled:
  - activity: 2.5.2
  - admin_audit
  - dashboard: 7.0.0
  - encryption
  - files_external: 0.7.0
  - files_readmemd: 1.2.2
  - survey_client: 1.8.0
  - suspicious_login: 4.1.0
  - user_ldap
```

### Nextcloud Signing status

```shell
No errors have been found.
```

**Comment by grrosminet** [comment]
![Screenshot_20220509_155028](https://user-images.githubusercontent.com/3082795/167427358-1e7d0baf-0346-42a0-a800-c8aad42804a2.png)

**Comment by szaimen** [comment]
Duplicate of https://github.com/nextcloud/server/issues/31915.
Will be fixed with 24.0.1
